src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
```

Comment:
I agree entirely with `from X import *`. This is going to cause us pain down the road when we get a name conflict.

I'm generally fine with `from X import Y`, but prefer them to be in the function namespace. In this case, I don't see any harm in moving all of the imports inside the function definition.